style.css

```diff
@@ -310,3 +345,64 @@ body {
   border-radius: 8px;
   color: #fff;
 }
+
+@media only screen and (min-width: 708px) {
+  .work {
+    display: flex;
+    max-width: 70%;
+    margin: auto;
+    justify-content: space-between;
+  }
+
+  header {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0%;
+  }
+
+  .desktop {
+    margin-left: 50px;
+  }
+
+  .about-section {
+    display: flex;
+    width: 100%;
+  }
+
+  .about-left {
+    max-width: 40%;
+  }
+
+  .about-right {
+    flex-grow: 2;
+    margin-left: 50px;
+  }
+
+  main {
+    padding: 0%;
+    margin: 0%;
+  }
+
+  #nav-link {
+    display: flex;
+  }
+
+  nav > svg {
+    display: none;
+  }
+
+  #nav-link li {
+    padding-right: 20px;
+    list-style: none;
+  }
+
```

Comment:
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided.
- Your design:
<img width="704" alt="image" src="https://user-images.githubusercontent.com/77190203/160141631-1a1d8d6b-0236-4987-b9d0-d7e58cedceef.png">

- Figma design:
<img width="432" alt="image" src="https://user-images.githubusercontent.com/77190203/160141744-b9d44de2-3afd-4a45-ad2e-d329f6ae8c10.png">